pages/glossary/large-scale-text.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Large Scale Text
+key: large-scale-text
+---
+
+Text nodes with an [advance height](https://www.w3.org/TR/css-values/#length-advance-measure) of:
+
+- at least 18 [points](https://www.w3.org/TR/css-values/#pt), or
+- at least 14 [points](https://www.w3.org/TR/css-values/#pt) and the [font weight](https://www.w3.org/TR/css-fonts-3/#font-weight-prop) is `bold`, `bolder` or a number of 700 or higher.
```

Comment:
font weight > font weight property

Reply:
Can you explain why you think that's necessary? I don't think adding "property" explains anything that isn't already clear from linking to the spec.